mappings/net/minecraft/packet/handshaking/HandshakeC2S.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_118 net/minecraft/packet/handshaking/HandshakeC2S
```

Comment:
maybe the package name should be `handshake` 🤔

Reply:
Yeah, that's what I thought. The official state is called "handshaking" but "handshake" does sound better